src/main/java/org/jmhsrobotics/frc2024/Robot.java

```diff
@@ -11,6 +11,7 @@
 import org.jmhsrobotics.warcore.util.BuildDataLogger;
 import org.littletonrobotics.urcl.URCL;
 
+import edu.wpi.first.wpilibj.DataLogManager;
```

Comment:
Why the Additional import?

Reply:
there was an error without that import statement idk why